### DISCUSSION TITLE
Cheapest cost per CPU dedicated servers for fishnet

### DISCUSSION THREAD

**Discussion by jpeskov** [lichess-org/fishnet#230] [Ideas]
The main fishnet servers according to the [Lichess costs spreadsheet](https://docs.google.com/spreadsheets/d/1Si3PMUJGR9KrpE5lngSkHLJKJkb0ZuI4/preview) are $40/month at OVH and running the ancient [dual Xeon E5530 @ 2.40GHz, 5122 Passmark score **combined**](https://www.cpubenchmark.net/cpu.php?cpu=Intel+Xeon+E5530+%40+2.40GHz&id=1244&cpuCount=2). 

I was looking at the most cost-effective ($/CPU benchmark) dedicated servers in the same price range and came up with a few options based on the [Ryzen 5 3600, 17821 Passmark score](https://www.cpubenchmark.net/cpu.php?cpu=AMD+Ryzen+5+3600&id=3481).

- [Hetzner AX41 (34 EUR)](https://www.hetzner.com/dedicated-rootserver/matrix-ax)
- [Scaleway EM-A210R Elastic Metal (40 EUR)](https://www.scaleway.com/en/elastic-metal-a210r/)

Assuming 1:1 Stockfish:Passmark performance scaling, this would be over 300% the speed for roughly the same price monthly.

Anyone have any better leads?

_On the lowest end, I can report that Scaleway Stardust 1-CPU instances (~1.80 EUR/mo) do successfully run fishnet at 550 knps, though the CPU is not dedicated, may trigger acceptable use policy._

**Comment by niklasf** [comment]
Thanks! We have replaced `ushio` and indeed see ~2-3x performance for a similar price (despite fewer threads, due to higher clock speed and support for avx2). Now considering what to do with `cammy` and `adele` which additionally hold some data that we would have to find a new home for.

**Reply by niklasf** [reply]
We will likely retire the other two servers in about 1 month.

**Reply by jpeskov** [reply]
Cool, glad it's working out! 

The Hetzner AX41 does have a ["storage" option](https://www.hetzner.com/dedicated-rootserver/ax41) for the same price that has 2x2TB HDDs in RAID 1 (2TB usable).

**Comment by jpeskov** [comment]
Also, the [Hetzner AX101 @ 103 EUR](https://www.hetzner.com/dedicated-rootserver/ax101) has a Ryzen 9 5950X (32 threads), 128GB RAM, 3.84TB SSD. 

On paper, looks like a spec upgrade for half the price of `talos` ($228, 24 threads, 64GB RAM, 1.92TB SSD), but not sure of requirements for monitoring server.

**Reply by niklasf** [reply]
For fishnet, computing power is basically all that matters. For the other servers:
* We're happy to pay a premium for reliability, and we've had a much better expierence with OVH than with Hetzner.
* Most other services require or strongly benefit from a low-latency private network (called vRack at OVH). So we cannot easily mix hosters.
